tests/out/interpolate.ron.snap

```diff
@@ -0,0 +1,128 @@
+---
```

Comment:
I wonder if we can disable the IR snapshot and instead enable the MSL/SPV/GLSL backends, thus getting better coverage, and implicitly still testing IR

Reply:
Seems easy, I'll give it a try.